api/sheets/get_name.md

```diff
@@ -24,23 +24,23 @@ Before trying this method out, make sure to follow the setup directions in [the
 
 ### Example 1: Retrieve the name of a country by its DCID.
 
-![](/assets/images/sheets/sheets_get_name_cote_d_ivoire_cropped.png)
+![](../../assets/images/sheets/sheets_get_name_cote_d_ivoire_cropped.png)
```

Comment:
As mentioned before, please replace the old images, rather than add new filenames (like _cropped).

Also, the file paths should start with /assets/images, not ../../

This comment applies to all other images in these PRs

Reply:
Ok